docs/data-sources/audit_trail_event.md

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Audit Trail"
+page_title: "Scaleway: scaleway_audit_trail_event"
+---
+
+# scaleway_audit_trail_event
+
+Use this data source to get a list of existing Audit Trail events.
+For more information refer to the [Audit Trail API documentation](https://www.scaleway.com/en/developers/api/audit-trail/).
+
+## Example Usage
+
+```hcl
+# Retrieve all audit trail events on the default organization
+data "scaleway_audit_trail_event" "find_all" {
+  region = "fr-par"
```

Comment:
I'm not sure to understand why we want a region filter here if we want all the events in the organization. Will it filter event not happening on `fr-par`

Reply:
Nice catch, this region and the others in the doc are just oversights, i cleared them :heavy_check_mark: